server/views/pages/contacts/manage/contactConfirmation/basicDetails.njk

```diff
@@ -15,6 +15,15 @@
             classes: 'confirm-name-value'
         }
     },
+    {
+        key: {
+            text: "Gender"
+        },
+        value: {
+            text: (contact.genderDescription | capitalizeFirstLetter) if contact.genderDescription else 'Not provided',
```

Comment:
I'm a bit confused why we are doing the capitalise first letter. We should just use the reference data and update that if not happy with the format.

Reply:
Yes, I didn't notice there was `getReferenceDescriptionForCode` that could be used from the `referenceDataService`.
I have created a method called `formattedGender` like `formattedFullName` and pass it to view. 
I have added the same logic to both confirmation and contact details

Reply:
hey, I think there must be some crossed wires. `genderDescription` already has the reference code description in it. There are 2 fields, `gender`, which is the reference data code, and `genderDescription` which is the reference data description.

Reply:
gender displayed using `geneder.genderDescription` on the view